packages/documentation/docs/controls/input.mdx

```diff
@@ -2,14 +2,18 @@
 title: Input (text)
 ---
 
+import DeprecatedTags from '@site/src/components/Tags';
 import DocsTabs from '@site/src/components/DocsTabs';
```

Comment:
Should we use DocsTabs or LinkableDocsTabs like in forms-validation.mdx?

Reply:
This will be done in a separate task.